src/IconPacks.Avalonia.BoxIcons/PackIconBoxIcons.xaml

```diff
@@ -4,30 +4,31 @@
                     x:ClassModifier="internal">
 
     <Design.PreviewWith>
-        <iconPacks:PackIconBoxIcons Kind="RegularAtom" />
+        <iconPacks:PackIconBoxIcons Kind="RegularClinic" />
     </Design.PreviewWith>
 
-    <ControlTemplate x:Key="IconPacks.Avalonia.PackIconBoxIcons.Template" TargetType="iconPacks:PackIconBoxIcons">
+    <ControlTemplate x:Key="IconPacks.Avalonia.Template" TargetType="iconPacks:PackIconControlBase">

```

Comment:
Why removing the PackIconBoxIcons specific type and key name (here and below for the ControlTheme)?

Reply:
I think it's not needed as this is now more or less one-fits-all unless you see anything that is not covered with pseudo-classses

to add:
- :is-filled
- :is-sroked

Reply:
🙈 you are right, sure